src/tree/SubscriptionTreeItem.ts

```diff
@@ -118,7 +118,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         const newStaticWebAppName: string = nonNullProp(wizardContext, 'newStaticWebAppName');
 
         if (!context.advancedCreation) {
-            wizardContext.newResourceGroupName = newStaticWebAppName;
+            wizardContext.newResourceGroupName = await StaticWebAppNameStep.getRelatedName(wizardContext, newStaticWebAppName);
```

Comment:
You're supposed to just do `await wizardContext.relatedNameTask;` here

Here's the app service example:
https://github.com/microsoft/vscode-azureappservice/blob/main/src/tree/SubscriptionTreeItem.ts#L113